### PR TITLE
Add a test that moving while the client is syncing don't upload

### PR DESCRIPTION
**Pull request by ogoffart**
This is a test for the client issue https://github.com/owncloud/client/issues/4370

### CODE REVIEW

File: lib/owncloud/test_moveFilesTwice.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+nfiles = 20
+TEST_FILES = ['test%02d.dat'%i for i in range(nfiles)]
+
+def getFileId(syncdir, fileName):
+    return subprocess.check_output(["sqlite3" , syncdir + "/.csync_journal.db",
+        "select fileid from metadata where path = \"" + fileName + "\""])
+
+@add_worker
+def workerA(step):
+    if compare_client_version('2.1.0', '<='):
```
Comment (nickvergessen) [inline lib/owncloud/test_moveFilesTwice.py:22]: this will be executed with 2.1.1beta1, is that correct?

Comment (ogoffart) [inline lib/owncloud/test_moveFilesTwice.py:22]: Beta1 still does not have the fix, but the current 2.1 branch has.  But there is no way to be more granular i think.

Comment (nickvergessen) [inline lib/owncloud/test_moveFilesTwice.py:22]: Well you can do:

``` py
if not compare_client_version('2.1.1beta1', '>'):
```

so if it is not higher then beta1 you skip it.

Anyway clients should be updated fairly fast and we wont test beta1 forever I guess.

**Comment by nickvergessen** [comment]
Tested and works on master, stable9 and stable8.2 with client v2.1.1, branch 2.1

I think we can/should merge, @PVince81

**Pull request merged**